source/_components/device_tracker.markdown

```diff
@@ -65,6 +65,7 @@ devicename:
   track: yes
   hide_if_away: no
 ```
+_(Note:  **devicename** refers to the detected name of the device.  For instance, **my_iphone** )_
```

Comment:
Please use a note box for this. For an example, see our documentation:
https://home-assistant.io/developers/documentation/create_page/#html